### PR TITLE
feat(ui): Grouping info improvements

### PR DESCRIPTION
**Pull request by matejminar**
Before:
Screenshot 2020-05-20 at 17 47 01

After:
Screenshot 2020-05-20 at 17 48 20

- grouping config field behind a feature flag
- border on collapsable frames
- question tooltips

Fixes JAVASCRIPT-22AF
Fixes JAVASCRIPT-22AE

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx [hunk 1]
```diff
@@ -39,12 +41,22 @@ class GroupVariant extends React.Component<Props, State> {
...
     if (variant.hash !== null) {
-      data.push([t('Hash'), variant.hash]);
+      data.push([
+        t('Hash'),
+        <TextWithQuestionTooltip key="hash">
+          {variant.hash}
+          <QuestionTooltip
+            size="xs"
+            position="top"
+            title="Events with the same hash are grouped together"
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx:56]:
```suggestion
            title={t("Events with the same hash are grouped together")}
```

File: src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx [hunk 2]
```diff
@@ -106,9 +155,9 @@ class GroupVariant extends React.Component<Props, State> {
           <ContributionIcon isContributing={isContributing} />
           {t('By')}{' '}
           {variant.description
-            .split(' ')
+            ?.split(' ')
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx:158]: Can `variant` be null?

**Review by billyvg — APPROVED** [review]
lgtm

File: src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx]: Good catch!

File: src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx (hunk 2 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/events/groupingInfo/groupingVariant.tsx:158]: There's this one edge case where the server can send `null` description.

**Pull request merged**